src/prx/main.py

```diff
@@ -479,6 +479,7 @@ def process(observation_file_path: Path, output_format="csv"):
         f"Starting processing {observation_file_path.name} (full path {observation_file_path})"
     )
     rinex_3_obs_file = converters.anything_to_rinex_3(observation_file_path)
+    rinex_3_obs_file = helpers.repair_with_gfzrnx(rinex_3_obs_file)
```

Comment:
Some files from Google Smartphone Decimeter Challenge do not have the `TIME OF LAST OBS` in the header. Calling `gfzrnx` early solves this issue.